lib/screens/verify_email.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:e_care_mobile/screens/otp_form.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'pulsating_button.dart';
+
+class VerifyEmail extends StatefulWidget {
+  const VerifyEmail({Key key}) : super(key: key);
+
+  @override
+  _VerifyEmailState createState() => _VerifyEmailState();
+}
+
+class _VerifyEmailState extends State<VerifyEmail> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        iconTheme: IconThemeData(
+          color: Colors.black,
+        ),
+        title: Text('Verification', style: TextStyle(color: Colors.black),),
+        leading: CircleAvatar(
+          backgroundColor: Colors.white,
+          radius: 10,
+
+          child:  IconButton(
+            icon: Icon(Icons.chevron_left),
+            onPressed: () => Navigator.pop(context),
+            color: Colors.black,
+          ),
+        ) ,
+
+        elevation: 0,
+        backgroundColor: Colors.white12,
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: SingleChildScrollView(
+          child: Container(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                SizedBox(
+                  height: 30,
+                ),
+                Container(
+                  child: PulsatingCircleIconButton(),
+                ),
+                SizedBox(
+                  height: 50,
+                ),
+                Container(
+                  child: Column(
+                    children: [
+                      Text('Verification Code',
+                      style: TextStyle(
+                        fontSize: 22,
+                        color: Colors.purple,
+                        fontWeight: FontWeight.w700,
+                      ),
+                      ),
+                      SizedBox(height:10),
+                      Text('We have sent the verification code to your Mobile Number',
```

Comment:
this should be 'we have sent the verification code to your email'

Reply:
that's true. I will correct it in my next push